addon/services/blog.js

```diff
@@ -0,0 +1,20 @@
+import Service from '@ember/service';
+import { getOwner } from '@ember/application';
+
+export default class BlogService extends Service {
+  title = this.configParam('title');
+  description = this.configParam('description');
+  logo = this.configParam('logo');
+  coverImage = this.configParam('coverImage');
+  coverMeta = this.configParam('coverMeta');
+  navigation = this.configParam('navigation');
+  twitter = this.configParam('twitter');
+  facebook = this.configParam('facebook');
+  host = this.configParam('host');
+  paginate = this.configParam('paginate');
+
+  configParam(param) {
```

Comment:
so this particular change has actually subtly changed the semantics of what's happening here 🤔  I think the closest thing to do would be for each of the config parameters to be getters that access the config

I know that right now it probably doesn't make too much of a difference because the config can't change dynamically (related to what we talked about in #114 ) but I'm wondering what your thoughts are about the subtle changes in the semantics?

Reply:
yes I agree and I vaguely had the same idea when I was looking at the code. I still think that the semantics have changed but we don't need to worry because there is no scenario where this really matters (that I can think of anyway)